plugins/auth/index.js

```diff
@@ -126,7 +135,7 @@ exports.register = (server, options, next) => {
                 key: pluginOptions.jwtPublicKey,
                 verifyOptions: {
                     algorithms: [ALGORITHM],
-                    maxAge: EXPIRES_IN
+                    maxAge: EXPIRES_IN + EXPIRES_BUFFER
```

Comment:
Sorry, I don't known as well, but `maxAge` for `verifyOption` never be changed?

Reply:
I will check this.

Reply:
I checked this.

According to [jsonwebtoken library](https://github.com/auth0/node-jsonwebtoken/blob/master/README.md#jwtverifytoken-secretorpublickey-options-callback) , `maxAge` is the maximum allowed age for tokens to still be valid. 
So, It should be the same value of `MAX_EXPIRES_IN (+ EXPIRES_BUFFER)`.

If `expiresIn` value in `generateToken()` is less than `maxAge`, JWT expires by `expiresIn` value.
And if `expiresIn` value in `generateToken()` is greater than `maxAge`,  it will become error.